docs/user_guide/how_to_interface_with_user_defined_model.md

```diff
@@ -13,19 +13,31 @@ Users may want to:
 
 This feature allows any custom model that conforms to OpenMDAO standards and uses the appropriate configuration interfaces to be used within the H2Integrate ecosystem.
 
-## Example: paper mill model
+## Examples
 
-To demonstrate this capability, we include a minimal example of a custom model: a **paper mill**. This example includes:
+The models in the following two examples use standard H2Integrate configuration utilities and OpenMDAO components.
+
+```{note}
+You can combine an H2Integrate model and a custom model for the same technology within a single analysis.
+```
+
+### Custom General Finance Model Example: simple lco finance model
+
+To demonstrate this capability, we include an example of a custom general finance model: a **simple lco calculation** found in [Example 08: Wind Electrolyzer Example](https://github.com/NREL/H2Integrate/tree/develop/examples/08_wind_electrolyzer/). This example includes:
+- A `SimpleLCOFinanceConfig`: configuration class that is populated with inputs specific to that finance model.
+- A `SimpleLCOFinance` general finance model that computes the levelized cost of a commodity
+
+
+### Custom Technology Model Example: paper mill model
+
+To demonstrate this capability, we include a minimal example of a custom technology model: a **paper mill**. This example includes:
 
 - A `PaperMillPerformance` model that converts electricity input to paper output.
 - A `PaperMillCost` model that estimates capital and operational expenditures.
-- A `PaperMillFinance` model that computes the levelized cost of paper production (LCOP).
+- A `PaperMillFinance` technology finance model that computes the levelized cost of paper production (LCOP).
```

Comment:
Did you modify the example as well? I'm not sure that was included as part of this PR.